crates/diesel_models/src/payment_intent.rs

```diff
@@ -24,11 +23,7 @@ pub struct PaymentIntent {
     pub description: Option<String>,
     pub return_url: Option<String>,
     pub metadata: Option<serde_json::Value>,
```

Comment:
Use pii secret value for metadata

Reply:
Should we have all the serde json types as Strict types in diesel and domain models?